src/panels/config/entities/entity-registry-id-enable.ts

```diff
@@ -0,0 +1,137 @@
+import {
+  html,
+  css,
+  LitElement,
+  TemplateResult,
+  property,
+  customElement,
+  PropertyValues,
+} from "lit-element";
+
+import "@polymer/paper-input/paper-input";
+import "../../../components/ha-switch";
+import {
+  ExtEntityRegistryEntry,
+  EntityRegistryEntryUpdateParams,
+  updateEntityRegistryEntry,
+} from "../../../data/entity_registry";
+import { HomeAssistant } from "../../../types";
+import { PolymerChangedEvent } from "../../../polymer-types";
+import { HaSwitch } from "../../../components/ha-switch";
+import { computeDomain } from "../../../common/entity/compute_domain";
+
+@customElement("ha-registry-id-enable")
```

Comment:
Can we describe the functionality that we can edit instead of what features it currently exists of? I can see icon being added to this… wouldn't want to change the name then.

```suggestion
@customElement("ha-registry-basics-editor")
```